ios/Exponent/Versioned/Core/UniversalModules/EXFacebook/EXScopedFacebook.m

```diff
@@ -74,7 +69,8 @@ - (void)initializeWithAppId:(NSString *)appId
                    rejecter:(UMPromiseRejectBlock)reject
 {
   _isInitialized = YES;
-  [super initializeWithAppId:appId appName:appName resolver:resolve rejecter:reject];
+  NSString *scopedFacebookAppId = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"FacebookAppID"];
```

Comment:
Do you think it makes sense to log an information or a warning (`UMLogWarn`, `UMLogInfo`) that we're overriding the app ID with our own?

Reply:
Good idea, adding that in here and in `initWithExperienceId`